modules/webxr/doc_classes/WebXRInterface.xml

```diff
@@ -153,6 +176,10 @@
 		</method>
 	</methods>
 	<members>
+		<member name="enabled_features" type="String" setter="" getter="get_enabled_features">
+			A comma-seperated list of the features that were successfully enabled by [method XRInterface.initialize] when setting up the WebXR session.
```

Comment:
```suggestion
			A comma-separated list of features that were successfully enabled by [method XRInterface.initialize] when setting up the WebXR session.
```

Reply:
Ah, except for this doc change. :-) Thanks - updated in my latest push